remoteit/desktop
# DESK-1061 DESK-1077 manage-subscription-link and certificates

### Description


- Support for https certificates for secure connections
- Manage subscription link created for IoT license
- Test UI (Alt-Shift Avatar Menu to enable)
-  Advanced section collapsable
- Switch graphQL APIs
- Change leave button
- Disable active page link
- Update cloud controlled icon

### Test plan



1. Sign in
2. Go to a secure (https) service
3. Click create connection 
4. You should see an address that is not an IP address
5. Click launch
6. You should see the secure web page without security warnings
7. You should be able to turn off the HTTPS certificate in the settings page

### Ticket



<https://remoteit.atlassian.net/browse/DESK-1061>
<https://remoteit.atlassian.net/browse/DESK-1077>
